vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListImagesController.java

```diff
@@ -18,14 +19,20 @@
     @Autowired
     private IImageService imageService;
 
+    @RequestMapping("/staff/images/list")
+    public String listSpacesWithoutNum(Model model) {
+        return "redirect:/staff/images/list/1?sort=creationDate&order=desc";
```

Comment:
sort and order parameters should be empty so it picks up whatever default is used in the other controller method